Identity/PresentationLayer/Controllers/FriendsController.cs

```diff
@@ -0,0 +1,58 @@
+﻿using BusinessLayer.DTOs;
+using BusinessLayer.Extentions;
+using BusinessLayer.Services;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System.Security.Claims;
+
+namespace PresentationLayer.Controllers
+{
+
+    [ApiController]
+    [Route("api/friends")]
+    public class FriendsController : ControllerBase
+    {
+
+        private readonly FriendsService _friendsService;
+
```

Comment:
create intreface for friends and all the other services and pass interface to controller